README.md

```diff
@@ -2,3 +2,7 @@
 
 VS Code language plugin for PostgreSQL isolation test spec files
 (based on [this specification](https://github.com/postgres/postgres/blob/master/src/test/isolation/README))
+
+## Installation
+
+You can use the [this Visual Studio Market link](https://marketplace.visualstudio.com/items?itemName=onlined.pgspec) to install this plugin
```

Comment:
It would be better if you remove "this" in "this Visual Studio Market link".

Reply:
makes sense